server/game/core/card/UpgradeCard.ts

```diff
@@ -40,20 +39,6 @@ export class UpgradeCard extends UpgradeCardParent {
         return this._parentCard;
     }
 
-    /** The card that this card is underneath */
-    public set parentCard(card: Card | null) {
```

Comment:
Did this get moved elsewhere or is it being handled in the mixins?

Reply:
It got accidentally left in, I meant to remove it in the upgrade PR. The intent is that outside code cannot change the attached card directly, it must use the `attach()` and `unattach()` methods which keep state consistent